### PR TITLE
feat(docs): add button group stories

### PR DESCRIPTION
**Pull request by alizedebray**
Since checkbox and Radio buttons are no longer a part of ng-bootstrap, they are now part of our bootstrap components thus: 
- the page moved from "NgBootstrap" to "Bootstrap" in the demo,
- the button group style adapted to properly show active buttons,
- a button group story was created in storybook.

### CODE REVIEW

**Comment by swisspost-devs** [comment]
Preview environments are getting posted here, as soon as they are ready!

File: packages/documentation/src/stories/components/button-group/button-group.stories.tsx [hunk 1]
```diff
@@ -0,0 +1,161 @@
...
+      control: {
+        type: 'select',
+        labels: {
+          ' btn-sm': 'Small',
+          ' btn-rg': 'Regular',
+          '': 'Medium',
+          ' btn-lg': 'Large',
+        },
+      },
+      options: [' btn-sm', ' btn-rg', '', ' btn-lg'],
```
Comment (gfellerph) [inline packages/documentation/src/stories/components/button-group/button-group.stories.tsx:34]:
```suggestion
      options: [' btn-sm', ' btn-rg', '', ' btn-lg'],
      table: {
        category: 'General'
      }
```

File: packages/documentation/src/stories/components/button-group/button-group.stories.tsx [hunk 2]
```diff
@@ -0,0 +1,161 @@
...
+    },
+    element: {
+      name: 'Button Elements',
```
Comment (gfellerph) [inline packages/documentation/src/stories/components/button-group/button-group.stories.tsx:37]:
```suggestion
      name: 'Variant',
```

File: packages/documentation/src/stories/components/button-group/button-group.stories.tsx [hunk 3]
```diff
@@ -0,0 +1,161 @@
...
+      description: "Defined the semantic elements used as button within the group.",
+      control: {
+        type: 'select',
```
Comment (gfellerph) [inline packages/documentation/src/stories/components/button-group/button-group.stories.tsx:40]:
```suggestion
        type: 'radio',
```

File: packages/documentation/src/stories/components/button-group/button-group.stories.tsx [hunk 4]
```diff
@@ -0,0 +1,161 @@
...
+        labels: {
+          button: 'Buttons',
+          link: 'Links',
+          checkbox: 'Checkboxes',
+          radio: 'Radio Buttons',
+        },
+      },
+      options: ['button', 'link', 'checkbox', 'radio'],
```
Comment (gfellerph) [inline packages/documentation/src/stories/components/button-group/button-group.stories.tsx:48]:
```suggestion
      options: ['button', 'link', 'checkbox', 'radio'],
      table: {
        category: 'General'
      }
```

File: packages/documentation/src/stories/components/button-group/button-group.stories.tsx [hunk 5]
```diff
@@ -0,0 +1,161 @@
...
+      },
+      control: {
+        type: 'inline-radio',
+        labels: {
+          1: 'First',
+          2: 'Second',
+          3: 'Third',
+        },
+      },
+      options: [1, 2, 3],
```
Comment (gfellerph) [inline packages/documentation/src/stories/components/button-group/button-group.stories.tsx:65]:
```suggestion
      options: [1, 2, 3],
      table: {
        category: 'Value'
      }
```

File: packages/documentation/src/stories/components/button-group/button-group.stories.tsx [hunk 6]
```diff
@@ -0,0 +1,161 @@
...
+      },
+      control: {
+        type: 'inline-check',
+        labels: {
+          1: 'First',
+          2: 'Second',
+          3: 'Third',
+        },
+      },
+      options: [1, 2, 3],
```
Comment (gfellerph) [inline packages/documentation/src/stories/components/button-group/button-group.stories.tsx:82]:
```suggestion
      options: [1, 2, 3],
      table: {
        category: 'Value'
      }
```

File: packages/documentation/src/stories/components/button-group/button-group.stories.tsx [hunk 7]
```diff
@@ -0,0 +1,161 @@
...
+    },
+    label_1: {
+      name: 'First Label',
+      description: 'Defines the label of the first button.',
+      control: {
+        type: 'text',
+      },
```
Comment (gfellerph) [inline packages/documentation/src/stories/components/button-group/button-group.stories.tsx:89]:
```suggestion
      },
      table: {
        category: 'Content'
      }
```

File: packages/documentation/src/stories/components/button-group/button-group.stories.tsx [hunk 8]
```diff
@@ -0,0 +1,161 @@
...
+    },
+    label_2: {
+      name: 'Second Label',
+      description: 'Defines the label of the second button.',
+      control: {
+        type: 'text',
+      },
```
Comment (gfellerph) [inline packages/documentation/src/stories/components/button-group/button-group.stories.tsx:96]:
```suggestion
      },
      table: {
        category: 'Content'
      }
```

File: packages/documentation/src/stories/components/button-group/button-group.stories.tsx [hunk 9]
```diff
@@ -0,0 +1,161 @@
...
+    },
+    label_3: {
+      name: 'Third Label',
+      description: 'Defines the label of the third button.',
+      control: {
+        type: 'text',
+      },
```
Comment (gfellerph) [inline packages/documentation/src/stories/components/button-group/button-group.stories.tsx:103]:
```suggestion
      },
      table: {
        category: 'Content'
      }
```

**Review by gfellerph — CHANGES_REQUESTED** [review]
Awesome work! I'd categorize the controls in storybook and make it a little more obvious that there are variants with checkboxes and radios.

File: packages/documentation/src/stories/components/button-group/button-group.stories.tsx [hunk 10]
```diff
@@ -0,0 +1,161 @@
...
+
+const ButtonTemplate = (args: Args, index: number) => {
+  const position = index + 1;
+  const label = args[`label_${position}`];
+  switch (args.element) {
+    case 'checkbox': {
+      const isSelected = args.selected?.includes(position);
+      return [
+        <input type="checkbox" className="btn-check" id={`btncheck${position}`} autoComplete="off" defaultChecked={isSelected}/>,
+        <label className={`btn${args.size} btn-secondary`} htmlFor={`btncheck${position}`}>{label}</label>,
```
Comment (gfellerph) [inline packages/documentation/src/stories/components/button-group/button-group.stories.tsx:116]:
```suggestion
        <input type="checkbox" className="btn-check" id={`btncheck${position}`} autoComplete="off" defaultChecked={isSelected} key={`check${position}`} />,
        <label className={`btn${args.size} btn-secondary`} htmlFor={`btncheck${position}`} key={`checklabel${position}`}>{label}</label>,
```

File: packages/documentation/src/stories/components/button-group/button-group.stories.tsx [hunk 11]
```diff
@@ -0,0 +1,161 @@
...
+      ];
+    }
+    case 'radio': {
+      const isChecked = position === args.checked;
+      return [
+        <input type="radio" className="btn-check" name="btnradio" id={`btnradio${position}`} autoComplete="off" defaultChecked={isChecked}/>,
+        <label className={`btn${args.size} btn-secondary`} htmlFor={`btnradio${position}`}>{label}</label>
```
Comment (gfellerph) [inline packages/documentation/src/stories/components/button-group/button-group.stories.tsx:123]:
```suggestion
        <input type="radio" className="btn-check" name="btnradio" id={`btnradio${position}`} autoComplete="off" defaultChecked={isChecked} key={`radio${position}`} />,
        <label className={`btn${args.size} btn-secondary`} htmlFor={`btnradio${position}`} key={`radiolabel${position}`}>{label}</label>
```

**Review by gfellerph — CHANGES_REQUESTED** [review]
Can you also add a changeset with major changes for styles?

**Review by gfellerph — APPROVED** [review]

**Pull request merged**